internal/daysteps/daysteps.go

```diff
@@ -12,9 +18,52 @@ const (
 )
 
 func parsePackage(data string) (int, time.Duration, error) {
-	// TODO: реализовать функцию
+	// Разделяем строку
+	sepData := strings.Split(data, ",")
+
+	// Проверяем правильность разделения
+	if len(sepData) != 2 {
+		return 0, 0, fmt.Errorf("Не удалось обработать данные")
+	}
+
+	// Преобразуем количество шагов в число
+	stepsCount, errSteps := strconv.Atoi(sepData[0])
+	if errSteps != nil {
+		return 0, 0, fmt.Errorf("Ошибка преобразования числа шагов: %s. Текст ошибки: %s", sepData[0], errSteps)
```

Comment:
При оборачивании нужно использовать `%w` (wrap), а просто при выводе в консоль `%v`
```suggestion
		return 0, 0, fmt.Errorf("Ошибка преобразования числа шагов: %s. Текст ошибки: %w", sepData[0], errSteps)
```

Reply:
Нужно везде поправить